pkg/gitreceive/build.go

```diff
@@ -112,6 +110,20 @@ func build(
 		}
 	}
 
+	_, disableCaching := appConf.Values["DEIS_DISABLE_CACHE"]
```

Comment:
I think my preference would be to do the same as `DEIS_KUBERNETES_DEPLOYMENTS` (I'm not sure how it's done there, but if we do it the same, we're consistent). Also, if you feel strongly about this, we might want to change both.